app/models/post.rb

```diff
@@ -0,0 +1,13 @@
+class Post < ActiveRecord::Base
+  belongs_to :user, class_name: 'User'
```

Comment:
- [ ] Kindly change the name of the association to match the ERD, all the tables where a user is a foreign_key should have `author_id` instead of `user_id`. In this case, this line should be like this `belongs_to :author, class_name: 'User'`.